src/components/overlay/overlay.html

```diff
@@ -0,0 +1,14 @@
+<article>
+    <h2 class="article-header">Overlay</h2>
+    <section class="docs-block">
+        <aside class="docs-block__info">
+            <h3 class="docs-block__header">Default</h3>
+        </aside>
+        <div class="docs-block__content docs-block__content--centered">
+            {% capture iframe_content %}
+                {% include components/overlay/example.html %}
+            {% endcapture %}
+            <iframe width="600" height="400" srcdoc='{{ iframe_content | escape }}'></iframe>
```

Comment:
that's a bit tricky. I didn't want to setup a separate task for copying all these example files that you guys were including via `<iframe>` because that would be messy. So what I'm doing here is inlining them. the output will be something like `<iframe srcdoc='<html><head>...<body>...'>`. From the user perspective nothing changed.